plugins/jekyll/src/main/kotlin/JekyllPlugin.kt

```diff
@@ -21,8 +22,8 @@ class JekyllPlugin : DokkaPlugin() {
     val renderer by extending {
         (CoreExtensions.renderer
                 providing { JekyllRenderer(it) }
-                applyIf { format == "jekyll" }
-                override plugin<DokkaBase>().htmlRenderer)
+                // TODO NOW: Is this correct?
```

Comment:
Good question, I think it is, @Kordyjan?

Reply:
I don't think it is correct as long as `hmtlRenderer` is in the base plugin. I may be missing something but it looks like after applying jekyll plugin you would end up with two loaded renderers.